docs/api/sm-api.md

```diff
@@ -15,6 +15,8 @@ Secure Messaging within vets.gov enables secure, non-emergency, communications b
 | GET /messaging/health/messages/:id                | Gets a message                      | None                          |
 | GET /messaging/health/messages/:message_id/thread | List messages in thread             | [Pagination](#pagination)     |
 | POST /messaging/health/messages                   | Sends a message.                    | [json payload](#message) |
+| POST /messaging/health/message_drafts             | Creates a draft                     | [json payload](#message) |
+| POST /messaging/health/message_drafts/:id         | Updates a draft                     | [json payload](#message) |
```

Comment:
So when updating a draft, we're not going to return an object are we? I think for an update we should maybe just be returning `head :no_content` -- Could we add a line down below and for the return object on update action to say that it will return status 204 with no body.